app/policies/pageflow/membership_policy.rb

```diff
@@ -98,7 +98,7 @@ def destroy_for_entry?
     end
 
     def destroy_for_account?
-      @user.admin? ||
+      (Pageflow.config.allow_multiaccount_users && @user.admin?) ||
         AccountPolicy.new(@user, @membership.entity).destroy_membership_on?
```

Comment:
The first part of the conditional appears to be redundant. `add_membership_on?` (which `destroy_membership_on?` delegates to) will always be true anyway if the first part of the conditional is true.

Reply:
It can still make sense from a perspective of asking questions that are inexpensive to answer early on. However, I agree that it makes more sense to settle on one layer where we answer this, as long as we have no performance issue here

Reply:
But `add_membership_on?` will short circuit under the exact same conditions. So there really is no performance win here.

Reply:
I agree. When I wrote this, it was more challenging to me than it is now to keep different layers of code in mind at the same time. In the meantime, learning happened